src/core/prism.ts

```diff
@@ -162,7 +162,13 @@ export class Prism {
 	 * Usually a language definition like `Prism.languages.markup`.
 	 * @returns The highlighted HTML.
 	 * @example
-	 * Prism.highlight('var foo = true;', 'javascript');
+	 * import { Prism } from './src/core/prism';
+	 * import javascript from './src/languages/prism-javascript';
+	 * const prism = new Prism();
+	 * prism.components.add(javascript);
+	 * prism.highlight('var foo = true;', 'javascript')
+	 * // Returns:
+	 * `<span class="token keyword">var</span> foo <span class="token operator">=</span> <span class="token boolean">true</span><span class="token punctuation">;</span>`
```

Comment:
I wonder if it makes sense also to use modern JS in examples, like here, to use `let` instead of `var`?